src/sql-parser/src/parser.rs

```diff
@@ -2463,6 +2463,15 @@ impl<'a> Parser<'a> {
         self.expect_keyword(SOURCE)?;
         let if_not_exists = self.parse_if_not_exists()?;
         let name = self.parse_item_name()?;
+
+        // Webhook Source, which works differently than all other sources.
+        if self.peek_keywords(&[FROM, WEBHOOK]) {
```

Comment:
Prepositions are hard. I will think about this, as well, but `FROM` _feels_ wrong here to me. `AS` makes a little more sense but I'm not convinced. If you come up with an argument one way or another, lmk.

Reply:
I agree, `FROM` does feel weird here, but the reason I think we should stick with `FROM` is to be consistent with other sources. IMO it would be even _more_ weird if `WEBHOOK` was the only source that used `AS` instead of `FROM`. But I'm totally open to using `AS` if that's the consensus